src/Item.js

```diff
@@ -179,9 +179,13 @@ class Item extends EventEmitter {
 
     // if the item was saved with a custom inventory hydrate it
     if (this.inventory) {
+      // handle equipped containers.
+      if (!(this.inventory instanceof Inventory)) {
```

Comment:
Yeah I'm more interested in how this happened than just putting a band-aid on it. If you want to list repro steps I can try to reproduce it when I get home.